libs/portal-framework-ui/src/components/data-table/BaseTableStackedLayout.tsx

```
@@ -0,0 +1,240 @@
+                        </div>
+                      </div>
+                    );
+                  }
+                  return null;
+                })()}
+
+              <CardContent
+                className={cn("space-y-3", "[word-break:break-word]")}>
+                {row.getVisibleCells().map((cell) => {
+                  // Use the shared hook for mobile column hiding
+                  const isHidden = useMobileColumnHiding({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)